src/util/datablock/datablock.c

```diff
@@ -72,93 +73,132 @@ void _DataBlock_AddBlocks(DataBlock *dataBlock, size_t blockCount) {
     dataBlock->itemCap = dataBlock->blockCount * BLOCK_CAP;
 }
 
+void static inline _DataBlock_MarkItemAsDeleted(const DataBlock *dataBlock, char* item) {
+    for(int i = 0; i < dataBlock->itemSize; i++) item[i] = DELETED_MARKER;
```

Comment:
This line could just be a memset, though the compiler probably makes that optimization anyway.

Reply:
👍